src/components/command-bar/commands/search/components/recent-searches/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { IconHistory16 } from '@hashicorp/flight-icons/svg-react/history-16'
+import { useCommandBar } from 'components/command-bar'
+import {
+	CommandBarButtonListItem,
+	CommandBarDivider,
+	CommandBarList,
+} from 'components/command-bar/components'
+import { RecentSearch, RecentSearchesProps } from './types'
+
+const RecentSearches = ({ recentSearches }: RecentSearchesProps) => {
+	const { inputRef, setCurrentInputValue } = useCommandBar()
+
+	if (!recentSearches || recentSearches.length === 0) {
+		return null
+	}
```

Comment:
⛏️ nit: Along with the other suggestions, this could be simplified if we always return the same type/
```suggestion
	if (recentSearches.length === 0) {
		return null
	}
```